tests/db_functions/math/test_log.py

```diff
@@ -0,0 +1,33 @@
+import math
+from decimal import Decimal
+
+from django.db.models.functions import Log
+from django.test import TestCase
+
+from ..models import DecimalModel, IntegerModel
+
+
+class LogTests(TestCase):
+
+    def test_decimal(self):
+        DecimalModel.objects.create(n1=Decimal('12.9'), n2=Decimal('3.6'))
+        obj = DecimalModel.objects.annotate(n_log=Log('n1', 'n2')).first()
+        self.assertTrue(
```

Comment:
This looks difficult to debug.  If it fails all we'll see is something like "False is not True". Is there any way to formulate it using other assertions? If not, maybe add a comment about it? Adding an assertion helper naming something like `assertLogEqual` could reduce repetition.

Reply:
Thank you for the suggestion! I added a helper function assertLogEqual as you suggested.